app/assets/javascripts/discourse/app/mixins/card-contents-base.js

```diff
@@ -69,7 +74,7 @@ export default Mixin.create({
       post,
     });
 
-    this._showCallback(username, $target);
+    this._showCallback(username, $(target));
```

Comment:
The callbacks rely on this being jquery........ How deep down the d-jquerying rabbithole shall I go

Reply:
I think we can leave this one for now. We only intend to improve a little bit each time, not do it exhaustively!